src/index.html

```diff
@@ -20,8 +21,369 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="body">
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="Moyo_logo"
+          class="header__logo"
+        />
+      </a>
 
-  <body>
-    <h1>Catalog</h1>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--is-active is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
```

Comment:
The `href` attribute is set to `#`, which means this link does not lead anywhere. If this link is supposed to navigate to a different section or page, you should provide the correct URL or path.